src/controllers/AbstractEntityController.php

```diff
@@ -263,10 +264,11 @@ protected function view(): Response
             'timestampInfo' => $this->Entity->getTimestampInfo(),
         );
 
-        // SHOW RELATED ITEMS AND EXPERIMENTS
+        // RELATED ITEMS AND EXPERIMENTS
         if ($this->Entity->type === 'items') {
-            $renderArr['relatedItemsArr'] = $this->Entity->Links->readRelated('items');
-            $renderArr['relatedExperimentsArr'] = $this->Entity->Links->readRelated('experiments');
+            ['items' => $renderArr['relatedItemsArr'],
```

Comment:
make sure to run `yarn phpcs`

Reply:
`yarn phpcs` did not complain about this line.
As you may realize I'm a newbie. Thanks for your patience.